docs/RunningOnDeviceIOS.md

```diff
@@ -24,16 +24,10 @@ In Xcode, select your phone as build target and press "Build and run"
 
 You have built a great app using React Native, and you are now itching to release it in the App Store. The process is the same as any other native iOS app, with some additional considerations to take into account.
 
-### Disabling the developer menu
+### Using the offline bundle & Disabling the developer menu
```

Comment:
@JoelMarcey I think it could be more nice title, like ```Building an app for production```. or ```Building an app for distribution```.
```Using the offline bundle``` or ```Disabling the developer menu``` is not enough I think. 

The ```Using the offline bundle``` is the aciton, and ```Disabling the developer menu``` is the result. and ```Building an app for distribution``` is what people really want to do.
Let's see this sentence, ```Building an app for distribution using the offline bundle, Disabling the developer menu```. At least this ordering looks better. So the title must be ```Building an app for distribution``` or similar things, I think. 

what about you?